infra/cifuzz/actions/entrypoint.py

```diff
@@ -17,6 +17,7 @@
 import sys
 
 # pylint: disable=wrong-import-position
+# pylint: disable=import-error
```

Comment:
just curious, what error is this suppressing?

Reply:
This error suppresses the 'can not find cifuzz module' error with pylint because of the path addition.

Reply:
Ack.